Secrets/KeysExample.swift

```diff
@@ -4,3 +4,9 @@ struct Keys {
         static let clientKey = ""
     }
 }
+
+func swiftLintTest() {
+    let someForceCast = NSNumber() as! Int
+    let colonOnWrongSide :Int = 0
+    let itsAllGood = 1 
```

Comment:
Trailing Whitespace Violation (Medium Severity): Line #11 should have no trailing whitespace: current has 1 trailing whitespace characters